features/admin/adding_products_to_a_product_bundle_slot.feature

```diff
@@ -0,0 +1,32 @@
+@managing_product_bundles
+Feature: Adding Products to Product Bundle Slots
+  In order to assign products to the slots of a product bundle
+  As an administrator
+  I want to be able to add products to the slot of a product bundle
+
+  Background:
+    Given the store has a product bundle "Hefty smurfs workout dress"
+    And this product bundle has a slot named "tights"
+    And this product bundle has also a slot named "hats"
+    And the store has a product "Hefty smurfs workout dress"
+    And this product is a product bundle product
```

Comment:
Perhaps it is good as it is, but I first stumbled across the duplicated initialization of the product bundle, i.e. creating the product bundle and the product bundle product, but then not explicitly associating both.
My personal opinion/suggestion would be to avoid duplicating and instead do the creation of the product in this step, too:
`
Given the store has a product bundle "Hefty smurfs workout dress"
`
(Of course we then have to manually push the newly created product to sharedStorage.)

Reply:
I agree. I also thought about this. I will shrink it together.